solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,151 @@
+"""Simulating an ATM. """
+
+import sys
+
+ATM_FILE = 1  # The number 1 refers to the place of the file as a parameter
+
+
+def check_the_balance(costumer_id, costumers_dictionary):
+    """Check the state of costumer's money in the bank.
+
+    Args:
+    costumer_id (string): costumer id.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    balance = int(costumers_dictionary[costumer_id][1])
+    return balance
+
+
+def cash_withdrawal(costumer_id, amount_to_withdrawal, costumers_dictionary):
+    """Withdrawal from the ATM amount of money.
+
+    Args:
+    costumer_id (string): costumer id.
+    amount_to_withdrawal (integer): amount the costumer wants to withdrawal.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    balance_now = check_the_balance(costumer_id, costumers_dictionary)
+    if amount_to_withdrawal > 0:
+        if balance_now >= amount_to_withdrawal:
+            new_balance = balance_now - amount_to_withdrawal
+            costumers_dictionary[costumer_id][1] = str(new_balance)
+            return True
```

Comment:
Make sure that everywhere in the code, and any code you write next to have blank line at the end of logic block (if, else, try, etc.)